conanfile.py

```diff
@@ -2,29 +2,42 @@
 
 class PahomqttclibConan(ConanFile):
     name = "pahomqttc"
-    license = "<Put the package license here>"
-    author = "<Put your name here> <And your email here>"
-    url = "<Package recipe repository url here, for issues about the package>"
+    version = "1.2.1"
     description = "This recipe file used to build and package binaries of pahomqttc repository"
     topics = ("<Put some tag here>", "<here>", "<and here>")
-    settings = "os", "compiler", "build_type", "arch"
-    options = { "shared": [True, False] }
-    default_options = "shared=False"
+    url = "https://github.com/elear-solutions/paho.mqtt.c"
+    license = "<Put the package license here>"
     generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+    }
+    default_options = {key: False for key in options.keys()}
+    default_options ["shared"] = False
 
-    def build(self):
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _configure_cmake(self):
         cmake = CMake(self)
         if (self.settings.os == "Android"):
             cmake.definitions["Platform"] = "android"
         cmake.configure(source_folder=".")
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
         cmake.build()
+        cmake.install()
 
     def package(self):
-        self.copy("*.h", dst="include", src="src")
-        self.copy("*", dst="lib", src="lib", keep_path=False)
+        self.copy("*.h", dst="include/pahomqttc", src="package/include/pahomqttc")
+        self.copy("*", dst="lib", src="package/lib", keep_path=False)
```

Comment:
kindly check the package folder inside the build folder to check if you are exporting all the required files.